examples/csv2argus.py

```diff
@@ -155,8 +179,8 @@ def parse_csv_into_metrics(csvfile):
             # abort without timestamp
             try:
                 ts = row[opts.timestampcolumn] and to_gmt_epoch(row[opts.timestampcolumn])
```

Comment:
This is still setting `ts` to `None` if a row has it blank for any reason. This could result in a cryptic error later.